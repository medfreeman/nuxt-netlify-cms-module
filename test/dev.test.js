import { get, commonBefore, commonAfter } from "./nuxt";

describe("module dev mode", () => {
  beforeAll(async () => {
    await commonBefore({ dev: true });
  });

  afterAll(async () => {
    await commonAfter();
  });

  test("render", async () => {
    const html = await get("/");
    expect(html).toContain("Works!");
  });

  test("admin", async () => {
    const html = await get("/admin/");
    expect(html).toMatchSnapshot();
  });
});
