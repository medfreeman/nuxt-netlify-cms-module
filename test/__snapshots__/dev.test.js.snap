// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`module dev mode admin 1`] = `
"<!DOCTYPE html>
<html>
  <head>
    <meta charset=\\"utf-8\\" />
    <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0\\" />
    <link href=\\"/admin/config.yml\\" type=\\"text/yaml\\" rel=\\"cms-config-url\\">
    <title>Content Manager</title>
  </head>
  <body>
  <script type=\\"text/javascript\\" src=\\"/admin/bundle.95871937146c00e3a6dc.js\\"></script></body>
</html>
"
`;
